gui/src/logs.h

```diff
@@ -0,0 +1,57 @@
+#ifndef LOGS_H_
+#define LOGS_H_
+
+#include <QDir>
+#include <QFile>
+#include <QStandardPaths>
+#include <QString>
+#include <QTextStream>
+
+static QString log_path = "";
+
+void logOutputToFile(QtMsgType type, const QMessageLogContext &context,
+                     const QString &msg) {
+  if (log_path.isEmpty()) {
+    // log_path assignment needs to be delayed until first usage of this
+    // function, so it can find the right app folder.
+    log_path =
```

Comment:
I vaguely remember the question of path being raised in some channel but then I guess we neglected to have that discussion. Where exactly will this end up? Is it not standard for apps in Linux to keep their logs somewhere under `/var/log` ?

Reply:
Looks like AppLocalDataLocation is evaluated to `"~/.local/share/<APPNAME>", "/usr/local/share/<APPNAME>", "/usr/share/<APPNAME>"` on linux according to the [docs](https://doc.qt.io/qt-5/qstandardpaths.html). Would the GUI have root permissions to write logs into `/var/log`?

Reply:
Ok, I guess this makes sense. No, we shouldn't give it root permissions. Let's keep it there, except maybe add something to a readme about where these logs are to be found.